DependencyInjection/SonataDashboardExtension.php

```diff
@@ -57,6 +58,11 @@ public function registerParameters(ContainerBuilder $container, array $config)
         $container->setParameter('sonata.dashboard.admin.block.entity', $config['class']['block']);
         $container->setParameter('sonata.dashboard.admin.dashboard.entity', $config['class']['dashboard']);
 
+        $container->setParameter(
+            'sonata.dashboard.admin.dashboard.templates.compose', $config['templates']['compose']);
+        $container->setParameter(
+            'sonata.dashboard.admin.dashboard.templates.compose_container_show', $config['templates']['compose_container_show']);
```

Comment:
Dunno how this passed validation @Soullivaneuh .

@stphnpt this is how it should be done : 

```
$container->setParameter(
    'sonata.dashboard.admin.dashboard.templates.compose',
    $config['templates']['compose']
);
```